Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:latest
+
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY package.json /usr/src/app/
+RUN npm install -g micro
+RUN npm install
+
+# Copy app source code
+COPY . /usr/src/app
```

Comment:
it would be better to copy the directory contents first, e.g.

```
#FROM...

# WORKDIR...

COPY . /usr/src/app # i think this can actually be "COPY . ." but in this case it's more clear to be explicit as you have it

RUN yarn install

CMD ["yarn", "start"]
```